packages/plugins/docs-ui/src/lib/components/links/document-links-panel.component.html

```
@@ -0,0 +1,83 @@
+			<button *ngIf="canLink" nbButton ghost size="tiny" (click)="attachExisting()">
+				<nb-icon icon="link-2-outline"></nb-icon>
+				{{ 'DOCS.LINKS.ATTACH_EXISTING' | translate }}
+			</button>
+
+			<ng-container *ngIf="canUpload">
+				<button nbButton ghost size="tiny" [disabled]="uploading" (click)="fileInput.click()">
+					<nb-icon icon="upload-outline"></nb-icon>
+					{{ 'DOCS.LINKS.UPLOAD_NEW' | translate }}
+				</button>
+				<input
+					#fileInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)